Tests/InstrumentationTests/NetworkStatusTests/NetworkStatusTests.swift

```diff
@@ -0,0 +1,152 @@
+// Copyright © 2021 Elasticsearch BV
```

Comment:
is the different copyright intentional?

Reply:
Oops! usually I catch this. I'll correct it right away.